src/_blog/libp2p-IPFSþing-2023-recap.md

```diff
@@ -0,0 +1,137 @@
+
+<div class="container" style="display:flex; column-gap:10px;">
+    <figure>
+        <img src="../assets/ipfs-thing-2023-03.jpg"  width="500">
+        <figcaption style="font-size:x-small;">Group shot of the IPFSþing 2023</a>
+        </figcaption>
+    </figure>
+</div>
+
+Over the course of five days the the libp2p community gave 4 different talks on the recent libp2p developments and finished strong with a workshop where participants built their own peer-to-peer chat application leveraging the same technology as the [Universal Connectivity](https://github.com/libp2p/universal-connectivity) demonstrator project.
+
+## Goals
+
+The goals of the libp2p contributors attending IPFSþing were:
+
+1. Build excitement by demonstrating the [Universal Connectivity application](https://github.com/libp2p/universal-connectivity).
+2. Give updates on the continually improving performace, dealing with non-uniform network topoligy with hole punching, and lowering the barriers to libp2p compatibility.
+3. Reconnect with community contributors and building up the greater libp2p community.
+
+## Recap of Talks
+
+### Hole Punching in the Wild
+
+Max Inden (rust-libp2p maintainer, Software Engineer at Protocol Labs)
+
+@[youtube](R-ToBsdlEk4)
+
+Learn from Max on how libp2p's hole punching mechanism works and what we've learned from the measurement campaign run last year.
+
+### libp2p Performance
+
+<div class="container" style="display:flex; column-gap:10px;">
+    <figure>
+    <figure>
+        <img src="../assets/ipfs-thing-2023-02.jpg" width="100%">
+        <figcaption style="font-size:x-small;">Max Inden and Marco Munizaga entertain us with a skit about libp2p performance.
+            <a href="https://app.iconik.io/share/assets/16fdde28-dd2c-11ed-9cf1-fe682d265912/?object_type=collections&object_id=3a266a5e-dd28-11ed-9faf-e667e71f3893&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzaGFyZV9pZCI6IjBkM2VhZDJjLWRkMzEtMTFlZC1iYzA3LTFhMGQ4ZDEzMTExNCIsInNoYXJlX3VzZXJfaWQiOiIwZDRmMjQwZS1kZDMxLTExZWQtYmMwNy0xYTBkOGQxMzExMTQiLCJleHAiOjE4Mzk0MjM4MjZ9.OjJFG2ItCyURSqwzvxmh6dyFVo7LHwdR2puBq0KAm00">[Pic credit.]</a>
+        </figcaption>
+    </figure>
+</div>
+
+Max Inden (rust-libp2p maintainer, Software Engineer at Protocol Labs)
+Marco Munizaga (go-libp2p and zig-libp2p maintainer, Software Engineer at Protocol Labs)
+
+@[youtube](2h9jth3nvJw)
+
+Watch Max and Marco talk about all of the performance related aspects of libp2p and what optimizations are in the latest versions. They give a great overview of how the Protocol Labs libp2p team does automated performance testing using terraform and two AWS servers on opposite ends of the United States.
+
+### The Incredible Benefits of libp2p + HTTP: A Match Made in Decentralized Heaven
```

Comment:
The text/video below doesn't match this title.

That's a separate talk, so it looks like we're missing an entry maybe?

Reply:
fixed